File: Ruby/FizzBuzz/fizzbuzz.rb
n = 15
i = 1
while i <= n
  if i % 15 == 0
    puts "FizzBuzz"
  elsif i % 5 == 0
    puts "Fizz"
  elsif i % 3 == 0
    puts "Buzz"
  else
    puts i
  end
  i += 1
end

File: Elixir/FizzBuzz/fizzbuzz.ex
defmodule FizzBuzz do
  def say(n) do
    Enum.map_join 1..n, fn x ->
      cond do
        rem(x, 15) === 0 -> "FizzBuzz\n"
        rem(x, 3) === 0 -> "Fizz\n"
        rem(x, 5) === 0 -> "Buzz\n"
        true -> Integer.to_string(x) <> "\n"
      end
    end
  end
end

IO.puts FizzBuzz.say(15)
